DESCRIPTION.md

```diff
@@ -12,6 +12,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
   - Added the ability to read Snowflake's central configuration file.
   - Improved OCSP response caching to remove tmp cache files on Windows.
   - Added a parameter `server_session_keep_alive` in `SnowflakeConnection` that skips session deletion when client connection closes.
+  - Add exception handling to `test.integ.test_connection.test_drop_create_user` when executing `grant all on database to role` as the grant is partially executed.
```

Comment:
I think we don't need to include this in the release notes.

Reply:
Thanks for adding this label for me. I have removed it.